Sources/ZcashLightClientKit/Rust/ZcashRustBackend.swift

```diff
@@ -920,7 +927,7 @@ struct ZcashRustBackend: ZcashRustBackendWelding {
     }
 
     @DBActor
-    func getWalletSummary() async throws -> WalletSummary? {
+    func getWalletSummary(confirmationsPolicy: ConfirmationsPolicy = ConfirmationsPolicy()) async throws -> WalletSummary? {
```

Comment:
```suggestion
    func getWalletSummary(confirmationsPolicy: ConfirmationsPolicy = ConfirmationsPolicy.defaultTransferPolicy()) async throws -> WalletSummary? {
```
(appropriately wrapped).